cards.js

```diff
@@ -98,6 +102,24 @@ class CardSet {
     return cardSet;
   }
 
+  shuffle() {
+    for (let i = this.cards.length - 1; i > 0; i--) {
```

Comment:
```suggestion
    for (let i = this.cards.length - 1; i >= 0; i--) {
```
Feel free to correct me if i'm wrong bet man šķiet ka būtu jābūt šādi

Reply:
jo citādi neiziet cauri cards[0]

Reply:
Nē, nav nepieciešams, tā būtu viena lieka mainīšana, jo 0to elementu jau katru reizi ir varbūtība iemainīt. Paskaties algoritmu https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle

Reply:
nu nez pēc tās loģikas varam neņemt arī 1o elementu

Reply:
ja tomēr vajadzētu ņemt nebūs tik random kā vajadzētu

Reply:
ok i approve